website/docs/docs/introduction.md

```diff
@@ -11,6 +11,13 @@ The role of dbt within a modern data stack is discussed in more detail [here](ht
 
 dbt also enables analysts to work more like software engineers, in line with the dbt [Viewpoint](viewpoint).
 
+### What is the difference between dbt Core, the dbt CLI and dbt Cloud?
+
+
+dbt Core is the software that takes a [dbt project](projects) (`.sql` and `.yml` files) and a command and then creates tables/views in your warehouse. dbt Core includes a command line interface (CLI) so that users can execute dbt commands using a terminal program. dbt Core is [open source](https://github.com/dbt-labs/dbt) and free to use.
```

Comment:
```suggestion
dbt Core is the software that takes a [dbt project](projects) (`.sql` and `.yml` files) and a command and then creates tables/views in your warehouse. You can execute dbt commands from the command line. dbt Core is [open source](https://github.com/dbt-labs/dbt) and free to use.
```

Reply:
suggs:

- technical accuracy. i don't think we technically ship an actual CLI, do we? 
- consistency. switched from second person (your) to third person (users) in the same paragraph.

Reply:
thank you so much @nghi-ly ! i don't know about the first bullet - i think it's part of core in some ways - but can I double check with you @jtcohen6 if the below sentence is correct or not:

_dbt Core includes a command line interface (CLI) so that users can execute dbt commands using a terminal program._

Reply:
@mirnawong1 : thanks for double-checking! i'm curious to know!

Reply:
> i don't think we technically ship an actual CLI, do we?

@nghi-ly Do you mean this in the sense that Bash, PowerShell, etc would qualify as "actual CLI" (shells)?

If so, fair point! dbt Core can be executed as an application on the Command Line. From reading how other tools describe themselves, I think it's idiomatically correct to say that "dbt Core includes a CLI," in the sense of being able to be used as a CLI app, without implying that dbt is a command line shell.

I'm happy with whatever language you decide on here

Reply:
thank you @jtcohen6 ! @nghi-ly i'll wait until after coalesce to push this through (unless i can do it now before the docs freeze).